src/ActionPreview.jsx

```diff
@@ -16,6 +16,10 @@ const DEFAULT_TABS = [{
 }]
 
 class ActionPreview extends Component {
+  static defaultProps = {
+    tabName: 'Action'
```

Comment:
Shouldn't it be `Diff` like in [default state](https://github.com/alexkuz/redux-devtools-inspector/blob/master/src/redux.js#L8)?